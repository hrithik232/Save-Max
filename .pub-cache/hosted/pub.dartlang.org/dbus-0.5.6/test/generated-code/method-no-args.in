<node>
  <interface name="com.example.Test">
    <method name="Hello"/>
  </interface>
</node>
