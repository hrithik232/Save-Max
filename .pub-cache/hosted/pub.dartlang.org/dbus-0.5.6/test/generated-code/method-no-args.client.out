import 'package:dbus/dbus.dart';

class ComExampleTest extends DBusRemoteObject {
  ComExampleTest(DBusClient client, String destination, DBusObjectPath path) : super(client, destination, path);

  /// Invokes com.example.Test.Hello()
  Future<void> callHello({bool noAutoStart = false, bool allowInteractiveAuthorization = false}) async {
    await callMethod('com.example.Test', 'Hello', [], replySignature: DBusSignature(''), noAutoStart: noAutoStart, allowInteractiveAuthorization: allowInteractiveAuthorization);
  }
}
